extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -266,11 +280,15 @@ export class FileConfigPage extends ImportPage {
 
 	private async populateSchemaDropdown(): Promise<boolean> {
 		this.schemaLoader.loading = true;
+
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
+		let connection = await azdata.connection.getConnection(connectionUri);
+		connection.databaseName = this.model.database;
+
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
 		const escapedQuotedDb = this.databaseDropdown.value ? `[${(<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]')}].` : '';
```

Comment:
At this point since we're connecting to the database I don't see the point in using the 3 part name here anyways so I'd suggest just getting rid of it and querying sys.schemas directly.